examples/neighbors/plot_lmnn_classification.py

```diff
@@ -0,0 +1,85 @@
+"""
+============================================================================
+Comparing Nearest Neighbors and Large Margin Nearest Neighbor Classification
```

Comment:
"with and without Large Margin Nearest Neighbor Classification" (see https://github.com/scikit-learn/scikit-learn/pull/10058#discussion_r259932667 ) (but should be already there if you mix this example with NCA's)

Reply:
Done. Regarding merging NCA and LMNN documentation, that involves quite some work. I'm not sure, if it even makes sense to do that now, if there is plans to do a separate metric learning module. Can anyone else comment?